src/javascript/app/pages/user/account/settings/two_factor_authentication.js

```diff
@@ -0,0 +1,116 @@
+const QRCode           = require('davidshimjs-qrcodejs');
+const Client           = require('../../../../base/client');
+const BinarySocket     = require('../../../../base/socket');
+const FormManager      = require('../../../../common/form_manager');
+const getPropertyValue = require('../../../../../_common/utility').getPropertyValue;
+const localize         = require('../../../../../_common/localize').localize;
+
+const TwoFactorAuthentication = (() => {
+    const form_id = '#frm_two_factor_auth';
+    const state = ['disabled', 'enabled'];
+    let current_state,
+        next_state,
+        qrcode; // eslint-disable-line
+
+    const onLoad = () => {
+        init();
+    };
+
+    const init = () => {
+        BinarySocket.send({ account_security: 1, totp_action: 'status'}).then((res) => {
+            $('#two_factor_loading').remove();
+
+            if (res.error) {
+                handleError('status', res.error.message);
+                return;
+            }
+
+            current_state = state[res.account_security.totp.is_enabled];
+            next_state    = state[+(!res.account_security.totp.is_enabled)].slice(0, -1);
+
+            $(`#${current_state}`).setVisibility(1);
+            $('#btn_submit').text(localize(next_state));
+            $(form_id).setVisibility(1);
+
+            FormManager.init(form_id, [
+                { selector: '#otp', validations: ['req', 'number', ['length', { min: 6, max: 6   }]], request_field: 'otp', no_scroll: true },
+                { request_field: 'account_security', value: 1 },
+                { request_field: 'totp_action', value: next_state },
+            ]);
+            FormManager.handleSubmit({
+                form_selector       : form_id,
+                fnc_response_handler: handleSubmitResponse,
+                enable_button       : true,
+            });
+
+            if (current_state === 'disabled') {
+                $('.otp-form-group').css('padding-left', '60px');
```

Comment:
how about adding a class that adds the padding-left instead of manipulating the css in js file, and then removing it on line 58?